src/component/contents/exploration/DraftEditorBlockNode.react.js

```diff
@@ -299,43 +249,39 @@ class DraftEditorBlockNode extends React.Component<Props> {
           child,
           blockRenderMap,
         );
+
         const elementProps = getElementPropsConfig(
           child,
           editorKey,
           offsetKey,
           blockStyleFn,
           customConfig,
         );
+
         const childProps = {
           ...this.props,
           tree: editorState.getBlockTree(key),
           blockProps: customConfig.customProps,
           offsetKey,
+          wrapperTemplate,
           block: child,
         };
 
-        acc.push(
-          React.createElement(
+        acc.push({
+          wrapperTemplate,
+          block: child,
+          element: React.createElement(
```

Comment:
Before I can give the block nodes to `wrapBlockNodes` I'm mapping it into an array that has this shape `{ wrapperTemplate, block, element }`. Not super confident about this design, lmk if you have a better idea.

Reply:
That sounds reasonable!